src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -473,6 +473,20 @@ $(document).ready(() => {
         showNewInstructorForm();
     });
 
+    $("[name='instructorisdisplayed']").change((e) => {
+        if ($(e.target).prop('checked')) {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('value', 'Instructor');
```

Comment:
Here, on page load, the default value of the input field (set as "Instructor") is visible. (which must be what you noticed). However, if the user unchecks and then checks the checkbox again, he will see blank field instead of the default value were it not for this line.